docs/content/posts/2025-09-18-release-kubernetes-operator-1.13.0.md

```diff
@@ -0,0 +1,90 @@
+---
+title:  "Apache Flink Kubernetes Operator 1.13.0 Release Announcement"
+date: "2025-09-18T18:00:00.000Z"
+authors:
+- fcsaky:
+  name: "Ferenc Csaky"
+aliases:
+- /news/2025/09/18/release-kubernetes-operator-1.13.0.html
+---
+
+The Apache Flink community is excited to announce the release of Flink Kubernetes Operator 1.13.0!
+The version brings a number of important fixes and improvements to both core and autoscaler modules.
+
+We encourage you to [download the release](https://flink.apache.org/downloads.html) and share your experience with the
+community through the Flink [mailing lists](https://flink.apache.org/community.html#mailing-lists) or
+[JIRA](https://issues.apache.org/jira/browse/flink)! We're looking forward to your feedback!
+
+## Highlights
+
+### Flink 2.1 Support
+
+The Flink Kubernetes Operator and Autoscaler 1.13.0 brings support for Flink 2.1 version.
+
+```yaml
+apiVersion: flink.apache.org/v1beta1
+kind: FlinkDeployment
+metadata:
+  name: basic-example
+spec:
+  image: flink:2.1
+  flinkVersion: v2_1
+...
+```
+
+### Structured YAML Configuration Support
+
+Starting with Flink Kubernetes Operator 1.13.0, workloads running on Flink 1.20 and later can define their `flinkConfiguration` in a structured YAML format,
+consistent with how this capability was introduced in the core Apache Flink configuration. 
```

Comment:
This feature is not restricted to Flink 1.20 and above, it works with all supported flink versions. It's somewhat independent of the underlying Flink yaml support. It's also a bit more flexible in the sense that users can mix and match structured and non-structured parts for seamless migration